test/udev-test.pl

```diff
@@ -1554,6 +1554,17 @@ sub udev_setup {
                 return 0;
         }
 
+        # check if we are permitted to create block device nodes
+        my $block_device_filename = $udev_dev . "/sda";
+        if (system("mknod", $block_device_filename, "b", "8", "0")) {
+                warn "unable to create $block_device_filename";
+                return 0;
+        }
+        else
+        {
+                unlink $block_device_filename;
+        }
```

Comment:
no need to have the else block btw. The unlink can be invoked directly here, as the if block returned early. In the interest of keeping indentation levels small, let's hence get rid of the else block, and place the unlink call in place directly.

Reply:
done